cmake/ProjectCompilerPostConfig.cmake

```diff
@@ -44,6 +44,7 @@ IF (${Trilinos_ENABLE_Kokkos})
         "${KOKKOS_SETTINGS} make -f ${KOKKOS_SRC_PATH}/cmake/Makefile.generate_cmake_settings CXX=${CMAKE_CXX_COMPILER} generate_build_settings")
   endif()
   include(${Kokkos_GEN_DIR}/kokkos_generated_settings.cmake)
+  install(FILES ${Kokkos_GEN_DIR}/kokkos_generated_settings.cmake DESTINATION lib/cmake/Kokkos)
```

Comment:
I think you want to change this to:

```
install(FILES ${Kokkos_GEN_DIR}/kokkos_generated_settings.cmake DESTINATION ${${PROJECT_NAME}_INSTALL_LIB_DIR}/cmake/Kokkos)
```

See:

* https://tribits.org/doc/TribitsBuildReference.html#setting-the-install-prefix

Make sure and test that to be sure but I think that is correct.

Reply:
I just made this change and tested it in 0668d98